cfgov/diversity_inclusion/forms.py

```diff
@@ -12,75 +12,62 @@
     'class': 'a-text-input a-text-input__full',
 }
 
-# This is needed to disable Django's default Textarea sizing.
-textarea_attrs = {
-    'rows': None,
-    'cols': None,
-}
-textarea_attrs.update(text_input_attrs)
-
-
 class VoluntaryAssessmentForm(forms.Form):
     institution_name = forms.CharField(
-        max_length=250,
+        label='Institution name',
```

Comment:
Setting this label explicitly isn't necessary, given the field name of `instutition_name`, but if you want to for clarity and consistency, that's cool.

Reply:
These label names might still be tweaked a little, so I'm going to leave the `label` for now for consistency and just in case it changes to something that wouldn't be great in code.